### ISSUE TITLE
v2.5.0 is busted

### ISSUE THREAD

**Issue by mlissner** [freelawproject/eyecite#139]
@overmode, I went ahead and merged your latest code and then tried to cut a release for you, but it had a few problems. If you look in the actions tab, a number of the CI checks failed. The good news is, this didn't push anything to pypi, but that's also the bad news.

Any chance you can take a look at these issues and see what it takes to fix them?

**Comment by overmode** [comment]
Hi @mlissner,
I had a look but it doesn't seem related to my code.

The linter-related issues are :

```Error: There was an error when attempting to execute the process '/home/runner/work/eyecite/eyecite/.venv/bin/flake8'. This may indicate the process failed to start. Error: spawn /home/runner/work/eyecite/eyecite/.venv/bin/flake8 ENOENT```

and 

```/home/runner/work/_temp/a7e0567d-d311-4d58-9678-d4f0fce33fcd.sh: /home/runner/work/eyecite/eyecite/.venv/bin/pylint-json2html: /home/runner/work/eyecite/eyecite/.venv/bin/python: bad interpreter: No such file or directory```

It looks like an environment error, but I don't have any clue on what happened.

The tests failed because of `fast_diff_match_path` 
```ModuleNotFoundError: No module named 'fast_diff_match_patch'```

And the rest is for installing poetry 
```
Run curl -O -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py
  File "get-poetry.py", line 1
    404: Not Found
             ^
SyntaxError: invalid syntax
Error: Process completed with exit code 1.
```

```
Run curl -O -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py
  File "/home/runner/work/eyecite/eyecite/get-poetry.py", line 1
    404: Not Found
    ^^^
SyntaxError: illegal target for annotation
```

**Comment by mlissner** [comment]
Ugh, what a lot of trouble! We'll get this ironed out. Thanks for looking.

**Comment by overmode** [comment]
Good luck ! :+1:

**Comment by mlissner** [comment]
Thanks @overmode. How much is this delay causing you trouble? Did you just patch your dependencies with a version from git or something or is this a blocker of some kind for you?

**Comment by overmode** [comment]
I'm currently patching it by pip installing the main branch directly, so all good, thanks for asking

**Comment by mlissner** [comment]
OK, great. We'll get to this soon, then. It's in the queue.

**Comment by mlissner** [comment]
I think this is all fixed, @overmode, but we did discover an issue with the reporters-db while fixing it. We downgraded reporters-db to a working version in `main` until we can figure out what the underlying bug is, so you'll want to make sure you're pulling the latest from main if you're using a pinned commit. 

Once we get that bug fixed in reporters-db, we'll release a new version of eyecite with the fix.
